send-transaction-service/src/transaction_client.rs

```diff
@@ -329,69 +311,16 @@ impl TpuClientNextClient {
     }
 
     #[cfg(any(test, feature = "dev-context-only-utils"))]
-    pub fn cancel(&self) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
It is a bit confusing that this change is on top of the previous version that you are removing.
Is it because you already committed the previous version?
It would be a bit easier to review if it would not contain changes that are not relevant.

Reply:
It is on top of master